### DISCUSSION TITLE
Delete an Alpine store

### DISCUSSION THREAD

**Discussion by abasse** [alpinejs/alpine#3256] [1. Help]
Hi all,

Is it possible to delete an Alpine store after initializing store, later in the code? 

Thanks,

André

**Comment by ekwoka** [comment]
`Alpine.store('storename', null)`

**Reply by abasse** [reply]
Thanks. I tried that, but this will cause an error
<img width="549" alt="image" src="https://user-images.githubusercontent.com/3475978/199909043-7d67b24f-e6f0-48d8-b032-9c4985bda9b7.png">

**Reply by ekwoka** [reply]
I'm not really sure what the use case is for deleting a store. Maybe some clarity on that can make the solution more obvious.

Can you not just put it to an empty object/array or trying or false?
